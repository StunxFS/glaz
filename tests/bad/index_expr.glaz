fn main() {
  _ = 2[1];
}
