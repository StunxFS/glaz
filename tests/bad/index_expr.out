tests/bad/index_expr.glaz:2:8: error: type `untyped_int` does not support indexing
     | 
   2 |   _ = 2[1];
     |       ^^^^
     = note: only `str`, pointers and arrays supports indexing
glaz error: could not compile `index_expr`, aborting due to previous error.
